MobileCenter/MobileCenter/Internals/Utils/MSKeychainUtil.m

```diff
@@ -3,20 +3,30 @@
 
 @implementation MSKeychainUtil
```

Comment:
Our rule is that there should be a new line before comment, not new line only before comment. No?